pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -376,11 +376,16 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 	}
 
 	var sgRules = []sgRule{
-		{podSgId, "0.0.0.0/0", -1, -1, rules.ProtocolTCP},
+		{podSgId, "0.0.0.0/0", 0, 0, rules.ProtocolTCP},
 		{masterSgId, openStackSvcCIDR, etcdPort, etcdPort, rules.ProtocolTCP},
 		{masterSgId, openStackSvcCIDR, apiPort, apiPort, rules.ProtocolTCP},
 	}
 
+	var decommissionedRules = []sgRule{
+		{podSgId, workerSubnet.CIDR, 0, 0, ""},
+		{masterSgId, openStackSvcCIDR, 2379, 2380, rules.ProtocolTCP},
```

Comment:
aren't these the etcd ports? shouldn't we remove other ranges not this one?

Reply:
[This rule from 4.3](https://github.com/openshift/cluster-network-operator/pull/465/files#diff-4ab6aa530ac077ca991bba003d6ea267L1035) got replaced by a [rule opening just 2379 in 4.4](https://github.com/openshift/cluster-network-operator/pull/465/files#diff-4ab6aa530ac077ca991bba003d6ea267R1043).

Reply:
right!